### DISCUSSION TITLE
Deprecated Docker Image

### DISCUSSION THREAD

**Discussion by taoistmiao** [ucb-bar/chipyard#2142] [General]
Hi, I'm wondering the rationale behind deprecating the docker image as stated in the [official doc](https://chipyard.readthedocs.io/en/stable/Chipyard-Basics/Initial-Repo-Setup.html#deprecated-pre-built-docker-image)

**Comment by jerryz123** [comment]
chipyard/dockerfiles exists, but is not maintained.
